sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -19,6 +19,21 @@ export function useFormioUtils() {
     return Utils.getValue(form.submission, componentKey);
   };
 
+  /**
+   * find the formio component  and set the value
+   * @param form form.
+   * @param componentKey component api/id from formio.
+   * @param value value that need to be set to the field
+   */
+  const setComponentValue = (
+    form: any,
+    componentKey: string,
+    value: any,
+  ): any => {
```

Comment:
Do we need to return something here. I believe that the method could be returning void in this case.